### PR TITLE
delsinrowe.ms

### PR DESCRIPTION
**Pull request by adambash**

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/buffs/delsinrowe.ms [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            'powers': array(
+                "/buff",
+                "/jump 0-5",
+                "/run 0-9",
+                "/haste 0-5"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 10 true",
+                "/effect ". @plr ." 21 9999 4 true",
+                "/shot give ". @plr ." solarenergybeam"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/delsinrowe.ms]: missing a comma

File: CommandHelper/LocalPackages/commands/buffs/delsinrowe.ms [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            'val': 'eyJ0aW1lc3RhbXAiOjE0ODczMDA5OTg2OTQsInByb2ZpbGVJZCI6IjQwODdiYzdmYzIxMzQ3ZGZiZjJkNDE2NTAwMzBiZTQ1IiwicHJvZmlsZU5hbWUiOiJVbHRyYU93bERDIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS9mYjZkYmFiZTk4YWQ2M2QzNTkyMDViNDQ2NmJhZjczY2ViMjZiMWI4ZGU5ODI0YjM4Y2FkMTQxNGQzNDcwIn19fQ==',
+            'trail': '',
+            'op': 'false',
+            'speech': "Just because something isn't normal doesn't mean it needs to be fixed.",
+            'powers': array(
+                "/buff",
+                "/jump 0-5",
+                "/run 0-9",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/delsinrowe.ms]: proof of super speed

File: CommandHelper/LocalPackages/commands/buffs/delsinrowe.ms [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+                "/haste 0-5"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 10 true",
+                "/effect ". @plr ." 21 9999 4 true",
+                "/shot give ". @plr ." solarenergybeam"   
+                "/give ". @plr ." elytra"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/delsinrowe.ms]: proof of elytra required

File: CommandHelper/LocalPackages/commands/buffs/delsinrowe.ms (hunk 2 above)
Comment (adambash) [inline CommandHelper/LocalPackages/commands/buffs/delsinrowe.ms]: http://infamous.wikia.com/wiki/File:InFAMOUS-Second-Son-Delsin-Running-Neon-Afterimages-001.jpg

Comment (adambash) [inline CommandHelper/LocalPackages/commands/buffs/delsinrowe.ms]: http://infamous.wikia.com/wiki/File:InFAMOUS-Second-Son-Delsin-Running-Neon-Afterimages-001.jpg

Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/delsinrowe.ms]: this should be run 3

**Pull request merged**